constitution.md

```diff
@@ -486,3 +486,42 @@ The Congress shall have power to enforce this article by appropriate legislation
 
 No law varying the compensation for the services of the Senators and Representatives shall take effect, until an election of Representatives shall have intervened.
 
+# Amendment XXVIII
+
+He who denied it, supplied it.
```

Comment:
I cannot support such brazen contracts by which fallacious conclusions are contrived to imply the guilt of a party before all evidence has been brought to the forefront. It is ludicrous!

If we are to fall to such levels, then we are practically saying that "All those whom have smelt it, are those whom have dealt it". I protest your amendment.